webapp/components/user_settings/user_settings_security.jsx

```diff
@@ -73,6 +73,15 @@ class SecurityTab extends React.Component {
         };
     }
 
+    componentWillUpdate() {
+        if ((this.state.passwordError !== '' || this.state.serverError) && (this.state.currentPassword !== '' || this.state.newPassword !== '' || this.state.confirmPassword !== '')) {
```

Comment:
When the component updates: If any errors are showing, and the password fields are not empty, set the errors to empty.